__tests__/api.test.js

```
@@ -302,5 +301,66 @@ describe('app', () => {
       })
     })
     
-    
+    describe("8. POST /api/reviews/:review_id/comments", () => {
+        test('should take an object in the form `{ inc_votes: newVote } and update it to new value', () => {
+          const userComment = {
+            username: "mallionaire",
+            body: "You see tracy medicine isnt an exact science"
+          };
+          const finalResult = {
+            comment_id: expect.any(Number),
+            author: "mallionaire",
+            body: "You see tracy medicine isnt an exact science",
+            review_id: 2,
+            created_at: expect.any(String),
+            votes: 0
+          }  
+          return request(app)
+            .post('/api/reviews/2/comments')
+            .send(userComment)
+            .expect(201)
+            .then(({ body }) => {
+              const { comment } = body;
+                  expect(comment).toEqual(finalResult)
+            })
+        })
+          test('404; handles incorrect id number', () => {
+            const userComment = {
+              username: "mallionaire",
+              body: "You see tracy medicine isnt an exact science"
+            };
+            return request(app)
+            .post('/api/reviews/50/comments')
+            .send(userComment)
+            .expect(404)
+            .then(({body: { msg }}) => {         
+              expect(msg).toBe('No review found for review_id: 50')
+            })
+          })
+          
+          test('404; handles incorrect packet info', () => {

```

missing fileds should return a 400 error